familyplanningmodule/src/main/res/drawable/site_xters_continue_button.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape>
+            <stroke android:width="1dp" android:color="#0292d5" />
+            <solid android:color="#0292d5" />
```

Comment:
Can you move these text-colors and text-color-hints to the `colors.xml` files and reuse them